HomeTask2/basicOp.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * evaluates the expression formed with binary operator and numbers
+ * @param {string} operation Any character from , '+','-','*','/'
+ * @param {number} value1 number 1
+ * @param {number} value2 number 2
+ * @example basicOp('+', 4, 7)
+ * 
+ * @returns {number} evaluation of the expression
+ */
+
+function basicOp(operation, value1, value2) {
+    return eval(value1 + operation + value2)
```

Comment:
It is not recommended to use eval function as it has security issues. Hence develop another approach that would perform the same.